roles/nginx/tasks/http.yml

```diff
@@ -1,19 +1,15 @@
 ---
-- name: make the domain {{ domain }} available
+- name: "make {{ domains | join(',') }} available"
   template:
     src: etc/nginx/sites-available/site-available.j2
-    dest: "/etc/nginx/sites-available/{{ domain }}"
-  register: available_sites
+    dest: "/etc/nginx/sites-available/{{ domains | first }}"
 
-- name: enable the domain {{ domain }}
+- name: "enable {{ domains | join(',') }}"
   file:
-    src: "/etc/nginx/sites-available/{{ domain }}"
-    dest: "/etc/nginx/sites-enabled/{{ domain }}"
+    src: "/etc/nginx/sites-available/{{ domains | first }}"
```

Comment:
This is a good example of when the ordering becomes significant. I think in most cases, the domains will be static - but I wanted to point it out.